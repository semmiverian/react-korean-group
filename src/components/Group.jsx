import React from 'react'
import Image from './Image'
import ApplicationContext from '../context'


export default function Group({logo, onClick, id}) {
  return (
    <ApplicationContext.Consumer>
      {
        theme => (
          <div className={`w-1/5 cursor-pointer ${theme.card} rounded shadow flex flex-row justify-center p-3`} 
              onClick={onClick}
              data-testid={id}
              >
          <Image logo={logo} />

        </div>
        )
      }
    </ApplicationContext.Consumer>
  )
}
