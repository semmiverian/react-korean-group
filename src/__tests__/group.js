import React from 'react'
import {render} from 'react-testing-library'
import Group from '../components/Group'
import 'jest-dom/extend-expect'
import 'react-testing-library/cleanup-after-each'

it('should match snapshot', () => {
  const {container} = render(<Group />)

  expect(container).toMatchSnapshot()
})