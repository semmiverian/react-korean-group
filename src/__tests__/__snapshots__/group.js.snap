// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div>
  <div
    class="w-1/5 cursor-pointer bg-white rounded shadow flex flex-row justify-center p-3"
  >
    <img
      class="cover h-48 w-48"
    />
  </div>
</div>
`;
